app.js

```diff
@@ -2,13 +2,19 @@ require('dotenv').config();
 const express = require('express');
 const connectDB = require('./db/connect');
 const logger = require('./logging/logger');
+const artistsRouter = require('./routes/artists');
 const artistRouter = require('./routes/artists');
+const eventsRouter = require('./routes/events');
+const eventRouter = require('./routes/events');
```

Comment:
Why are you importing './routes/events' twice?

Reply:
because in the challenge routes, one route has get/events and others have get/event. I tried to import all on the same line but it didn't work out . So i just did it like this. I knew you would talk about it

Reply:
Remove one of them. When we get to the `event`/`events` bridge, we will cross it.

Reply:
I also removed the one for artists/artist. Done on 31eec05